omod/src/main/java/org/bahmni/module/hip/web/controller/PatientController.java

```diff
@@ -50,13 +48,12 @@ ResponseEntity<?> getExistingPatients(@RequestParam(required = false) String pat
         Set<Patient> matchingPatients = existingPatientService.getMatchingPatients(locationUuid,phoneNumber,patientName,
                 Integer.parseInt(patientYearOfBirth), patientGender);
         if (matchingPatients.size() == 0) {
-            return ResponseEntity.ok().body(new ErrorRepresentation(new Error(
-                    ErrorCode.PATIENT_ID_NOT_FOUND, "No patient found")));
+            return ResponseEntity.ok().body(new HashMap<String, Object>(){{ put("validPatient",false); put("patientUuid", null);}});
```

Comment:
Instead of hardcoding response body in all place. you can use a separate class

Reply:
Separate class for response type alone?

Reply:
yeah. similar to what we have in other controllers https://github.com/BahmniIndiaDistro/openmrs-module-hip/blob/master/omod/src/main/java/org/bahmni/module/hip/web/controller/CareContextController.java

Reply:
Changes have been made